examples/options/src/track_finding.cpp

```diff
@@ -80,48 +79,38 @@ track_finding::operator finding_config() const {
     out.ptc_hypothesis =
         detail::particle_from_pdg_number<traccc::scalar>(m_pdg_number);
 
-    return m_config;
+    return out;
```

Comment:
Take note that some of the track finding options could not be set from the command line so far. This was a sneaky little bug. (Though not a super important one for the moment.)